src/main/java/seedu/address/logic/commands/AppendInteractionCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INTERACTION;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.interaction.Interaction;
+import seedu.address.model.person.Person;
+
+public class AppendInteractionCommand extends Command {
+    public static final String COMMAND_WORD = "appendInteraction";
```

Comment:
Likewise, maybe just change it to "interaction" (in lower case - standard with the other ```COMMAND_WORDS```)